### PR TITLE
project dependencies adapted

### PR DESCRIPTION
**Pull request by Ginxo**
**Thank you for submitting this pull request**

**referenced Pull Requests**: 
project-dependencies adapted from
original
to
modified

> note: in case you want to increase image size and you browser adds a dark background you always have the chance to inspect browser and change background color to `#ffffff`

* https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1539

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

File: .ci/project-dependencies.yaml [hunk 1]
```diff
@@ -60,14 +60,16 @@ dependencies:
   - project: kiegroup/openshift-drools-hacep
     dependencies:
       - project: kiegroup/droolsjbpm-integration
+      - project: kiegroup/drools
+      - project: kiegroup/jbpm
```
Comment (danielezonca) [inline .ci/project-dependencies.yaml:64]: Wrong change: `openshift-drools-hacep` was correct, `jbpm` and `drools` should be added to `droolsjbpm-integration`

Comment (Ginxo) [reply .ci/project-dependencies.yaml]: done

**Review by danielezonca — APPROVED** [review]
LGTM :+1:

**Review by mbiarnes — APPROVED** [review]
+1

**Review by mareknovotny — APPROVED** [review]

**Pull request merged**